openupgrade_scripts/scripts/survey/15.0.3.4/pre-migration.py

```diff
@@ -0,0 +1,13 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE survey_survey
+        SET active = Case when state in ('draft','open') THEN true ELSE false END
```

Comment:
@sang250399 các từ khóa nên viết hoa hết cho dễ đọc nhé
```suggestion
        SET active = CASE WHEN state in ('draft','open') THEN true ELSE false END
```